pkgs/build-support/php/builders/v1/build-composer-project.nix

```
@@ -89,6 +90,13 @@ let
           composerStrictValidation = previousAttrs.composerStrictValidation or true;
         });
 
+      # Projects providing a lockfile from upstream can be automatically updated.
+      passthru = previousAttrs.passthru or { } // {
+        updateScript =
+          previousAttrs.passthru.updateScript
+            or (if finalAttrs.composerRepository.composerLock == null then nix-update-script { } else null);
+      };
```

Can't we do this instead?

```suggestion
      passthru.updateScript = previousAttrs.passthru.updateScript or
(if finalAttrs.composerRepository.composerLock == null then nix-update-script { } else null);
```